aten/src/ATen/native/miopen/Conv_miopen.cpp

```diff
@@ -1141,6 +1245,24 @@ Tensor miopen_convolution_backward_bias(
 {
   TensorArg grad_output{ grad_output_t, "grad_output", 1 };
 
+  // TODO: Workaround since MIOpen does not support NHWC bias
+  std::vector<int64_t> discard_dims;
+  for( int i = 0; i < grad_output_t.dim(); i++ ) {
+      if(i != output_channels_dim ) {
+          discard_dims.push_back(i);
+      }
+  }
+
+  Tensor outputBias = at::squeeze( at::sum(grad_output_t, discard_dims, true) );
+  if( outputBias.dim() == 0 ) {
+      // always return a tensor of shape [_]
```

Comment:
How come?

Reply:
If the result is just 1 element, 1 dim, at::squeeze will make the result a scalar. There are tests in test_nn.py that check this return value against a tensor of shape, [1].